lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1361,16 +1461,21 @@ def _scrape_metadata(self) -> dict:
         return self.topology.as_dict()
 
 
-class RuleFilesProvider(Object):
-    """A 'prometheus_scrape' provider class for rule files only.
+class PrometheusRulesProvider(Object):
+    """Forward rules to Prometheus.
 
-    This class is for sending rules to prometheus, unlike :class:`MetricsEndpointProvider`, which
-    is used for configuring a scrape target (including rules).
-    This is useful for providing "free standing" rules.
+    This object may be used to forward rules to Prometheus. At present it only supports
+    forwarding alert rules. This is unlike :class:`MetricsEndpointProvider`, which
+    is used for forwarding both scrape targets and associated alert rules. This object
```

Comment:
```suggestion
    forwarding alert rules (support for recording rules is not yet implemented).
    In contrast to :class:`MetricsEndpointProvider`, which is used for forwarding both
    scrape job and associated alert rules, this object is only for forwarding 
    free-standing rules. This object
```

Reply:
I am fine with most of this suggestion with the only minor exception of the use of the phrase "free-standing rules". I would rather not invent such phraseology unless it is coming from a product specification. Are you ok with that ?

Reply:
I agree with you on this @balbirthomas. The less internal nomenclature we introduce, the better - at least from an onboarding perspective. In that case, maybe changing `free-standing rules` to `rules from the lma-rules charm` would make sense?

Reply:
Well we are going to rename the `lma-rules` charm, sooo... :D